apps/marketing/public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "",
```

Comment:
Consider setting `name` to `DeepDish` and omitting the `short_name` for now... or omitting both for the time being.